src/app/mock/logs.ts

```diff
@@ -0,0 +1,3 @@
+import { Log } from '../models/log';
```

Comment:
It appears `Log` is an [empty interface](https://github.com/Azure/kashti/blob/39f8dee9c974a237e5b5601d9fffbf73ec9889a4/src/app/models/Log.ts)

Do we want to add fields on the interface and make the mock data typed?

Reply:
The mock/api log data is a blob of plain text here; no structure - I wasn't sure how to do so without _some sort_ of an interface though, hence the inclusion of an empty one